acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -17,7 +17,7 @@
 import static tech.pegasys.ethsigner.tests.tls.support.CertificateHelpers.createJksTrustStore;
 
 import tech.pegasys.ethsigner.core.config.ClientAuthConstraints;
-import tech.pegasys.ethsigner.core.config.PkcsStoreConfig;
+import tech.pegasys.ethsigner.core.config.DownstreamTlsOptions;
```

Comment:
Can we generalise this stucture, and re-use it for both sides of EthSigner?

Reply:
I think not as the set up of TLS options for EthSigner as a server is bit different than the TLS options for EthSigner as a client. But I am open to suggestions.